### PR TITLE
[SDA-6439] Add red-hat-managed tag to roles and policies

### PR DESCRIPTION
**Pull request by hunterkepley**
[SDA-6439](https://issues.redhat.com/browse/SDA-6439)

### CODE REVIEW

File: pkg/aws/tags/tags.go [hunk 1]
```diff
@@ -46,3 +46,6 @@ const Environment = prefix + "environment"
 
 // AdminRole tags the role as admin (true/false)
 const AdminRole = prefix + "admin_role"
+
+// RedHatManaged tags the role as red_hat_managed
+const RedHatManaged = prefix + "red_hat_managed"
```
Comment (vkareh) [inline pkg/aws/tags/tags.go:51]: This should be `red-hat-managed` (hyphens instead of underscores)

Comment (vkareh) [reply pkg/aws/tags/tags.go:51]: Also, remove the `prefix`, since we want `red-hat-managed` to be the entire tag.

Comment (hunterkepley) [reply pkg/aws/tags/tags.go]: @vkareh Made the changes you requested, please feel free to resolve this thread if it looks good!

**Comment by vkareh** [comment]
This looks good, thanks!
Can you please squash/fixup these commits so that there's only a single atomic one?

**Comment by hunterkepley** [comment]
> This looks good, thanks! Can you please squash/fixup these commits so that there's only a single atomic one?

Squashed them, should be one commit now!

**Comment by hunterkepley** [comment]
> This looks good, thanks! Can you please squash/fixup these commits so that there's only a single atomic one?

@vkareh Squashed successfully, should be good to go now!

**Comment by hunterkepley** [comment]
Finished testing right before the merge, everything seems to be working fine. STS roles are created with `red-hat-managed` set to True. Also checked policies

**Pull request merged**